samtranslator/model/__init__.py

```diff
@@ -5,16 +5,16 @@
 
 from samtranslator.intrinsics.resolver import IntrinsicsResolver
 from samtranslator.model.exceptions import ExpectedType, InvalidResourceException, InvalidResourcePropertyTypeException
```

Comment:
`ExpectedType` can be removed from import?

Reply:
If this is a valid comment, we should instead fix `pylint` so that it complains.

Reply:
it is used at line 30